index.html

```diff
@@ -14,166 +14,171 @@
       rel="stylesheet"
       href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
     />
-    <link rel="stylesheet" href="styles/style.css" />
+    <link rel="stylesheet" href="/styles/style.css" />
 
     <title>Bikes for Refugees</title>
   </head>
 
   <body>
-    <div class="header">
+    <section class="header">
+      <a href="#">
+        <img
+          src="/images/bikes-for-refugees_logo.jpg"
+          alt="Logo for Bikes for Refugees Scotland"
+          class="header__logo"
+        />
+      </a>
+
+      <section>
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Home</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">About Us</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">News & Events</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Contact Us</a>
+          </li>
+        </ul>
+      </section>
+
+      <button class="Don">Donate Now</button>
+    </section>
+
+    <section class="content">
+      <section class="main">
+        <section class="alert">
+          You've helped us give <span class="text-highlight">72</span> bikes to
+          refugees and asylum seekers so far. <strong>Thank you!</strong>
+        </section>
+
+        <section class="hero">
+          <h1>Bikes for Refugees</h1>
+          <p>
+            Providing donated bikes and accessories to refugees and asylum
+            seekers in Scotland.
+          </p>
+          <button class="Don">Donate a bike today</button>
```

Comment:
You gave two buttons the same class! This is a problem because if I want to change something about only one of the buttons, i cannot! 
The space between this button and the rest of the test is too large! because you added a margin to that css! You have to change this class and add new css to this!
 
![image](https://github.com/CodeYourFuture/bikes-for-refugees/assets/118762745/8573d541-e41f-45a1-ba6f-596000f9fe2a)